src/components/social.js

```diff
@@ -9,29 +9,57 @@ module.exports = React.createClass({
     var locale = this.context.intl.locale;
     var twitterShareURL = 'https://twitter.com/share?url=' + appURL +'/' + locale + '/&text=' + encodeURIComponent(this.context.intl.formatMessage({id: 'i_donated_to_mozilla'}));
     var facebookShareURL = 'https://www.facebook.com/sharer/sharer.php?u=' + appURL + '/' + locale + '/';
+    var emailSubject = '';
+    var emailBody = '';
+    var emailShareURL = 'mailto:someone@example.com?subject='+ emailSubject +'&body='+ emailBody +'';
     return (
       <div className="share-page">
         <div className="container">
           <h3>
             <div>{this.context.intl.formatMessage({id: 'tell_your_friends'})}</div>
             <div>{this.context.intl.formatMessage({id: 'help_spread_the_word'})}</div>
           </h3>
-          <div className="row">
-            <div className="half" id="facebook">
+          {/*Baseline social shares with just Facebook and Twitter*/}
+          <div className="row baseline-social">
+            <div className="third" id="facebook">
               <div className="social-button"></div>
               <a href={facebookShareURL} target="_blank">
                 <i className="fa fa-facebook fa-4x"></i>
                 <div>facebook</div>
               </a>
             </div>
-            <div className="half" id="twitter">
+            <div className="third" id="twitter">
               <div className="social-button"></div>
               <a href={twitterShareURL} target="_blank">
                 <i className="fa fa-twitter fa-5x"></i>
                 <div>twitter</div>
               </a>
             </div>
           </div>
+          {/*Test where a share via email button is added along with Facebook and Twitter*/}
+          <div className="row social-with-email">
+            <div className="third" id="facebook">
+              <div className="social-button"></div>
+              <a href={facebookShareURL} target="_blank">
+                <i className="fa fa-facebook fa-2x"></i>
+                <div>facebook</div>
+              </a>
+            </div>
+            <div className="third" id="twitter">
+              <div className="social-button"></div>
+              <a href={twitterShareURL} target="_blank">
+                <i className="fa fa-twitter fa-2x"></i>
+                <div>twitter</div>
+              </a>
+            </div>
+            <div className="third email-share email-share-test" id="email">
+              <div className="social-button"></div>
+              <a href={emailShareURL} target="_blank">
+              <i className="fa fa-envelope fa-2x"></i>
+                <div>email</div>
```

Comment:
My previous comment is hidden, but we should be able to reuse the existing string and do:

`<div>{this.context.intl.formatMessage({id: 'email'})}</div>`